### DISCUSSION TITLE
Mouse Call Categories

### DISCUSSION THREAD

**Discussion by cmlenell1** [DrCoffey/DeepSqueak#150] [General]
Hi,
Has anyone created a mouse supervised clustering dataset that uses the ~10 standard call types? 
I would rather not start from scratch. 
Thanks!
Charlie

**Comment by Cesarmedina123** [comment]
> 

I have been trying to create one for some time now, but still no luck. If any one has had better luck than me, please, let me know as well.

Gracias,

Cesar

**Comment by DrCoffey** [comment]
Hey @cmlenell1 

Do you have some really good examples for each of the 10 call types? Something I have done before is to create a K-means model with only 1 or 2 calls per category that essentially define the 'ideal' call for each category. If you have 10 calls that each represent 1 category, tell DeepSqueak to K-means cluster calls into 10 categories, then rename them in the clustering GUI. each call contour will be the exact centroid for the model. Then apply that model to all the other files.

-Kevin

**Reply by cmlenell1** [reply]
Thanks for this suggestion! I will try this. The problem with fixed category systems is that within 1 category there tends to be several subcategories of calls (e.g., step down) so finding representative images is somewhat difficult. I feel like the main component of the call should also matter.... I  hope we crack the 'code' soon! Thanks again!

<img width="147" alt="stepdown2" src="https://user-images.githubusercontent.com/45463545/158198989-22367baa-cec6-4f14-8573-62995c61a6e0.png">
<img width="224" alt="stepdown1" src="https://user-images.githubusercontent.com/45463545/158198996-587a1e7f-b694-4117-9ea6-f2e8d6c1c0b1.png">
<img width="92" alt="stepdown4" src="https://user-images.githubusercontent.com/45463545/158198998-559e7537-ef80-44ce-b86c-0c38840100a9.png">
<img width="173" alt="stepdown3" src="https://user-images.githubusercontent.com/45463545/158199001-9884cd8b-c0c8-4c16-96fa-a6a421ee8279.png">
